SCons/Tool/gfortran.py

```diff
@@ -29,24 +29,27 @@
 selection method.
 """
 
-import SCons.Util
+from SCons.Util import CLVar
 
 from . import fortran
 
 
 def generate(env):
-    """Add Builders and construction variables for gfortran to an
-    Environment."""
+    """Add Builders and construction variables for gfortran."""
     fortran.generate(env)
 
-    for dialect in ['F77', 'F90', 'FORTRAN', 'F95', 'F03', 'F08']:
-        env[f'{dialect}'] = 'gfortran'
-        env[f'SH{dialect}'] = f'${dialect}'
-        if env['PLATFORM'] in ['cygwin', 'win32']:
-            env[f'SH{dialect}FLAGS'] = SCons.Util.CLVar(f'${dialect}FLAGS')
-        else:
-            env[f'SH{dialect}FLAGS'] = SCons.Util.CLVar(f'${dialect}FLAGS -fPIC')
+    # fill in other dialects (FORTRAN dialect set by fortran.generate(),
+    # but don't overwrite if they have been set manually.
+    for dialect in ['F77', 'F90', 'F95', 'F03', 'F08']:
+        if dialect not in env:
+            env[f'{dialect}'] = 'gfortran'
```

Comment:
Did you mean to drop the CLVar() ?

Reply:
It doesn't. The diff isn't really helping here.  The compiler names (old 43, 44; nww 45, 47) were/are not CLVar (single item).  Old line 46 is not needed, it's just been set by calling `fortran.generate()`, so I dropped it. Old line 48, new line 52 both do use CLVar.